acceptancetests/assess_caas_bootstrap.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+
+    Spining up k8s cluster and asserting the cluster is `healthy`
```

Comment:
It would be nice to quickly outline what 'healthy' means in this context.
Also typo, spining -> spinning.